packages/datetime/src/index.ts

```diff
@@ -14,6 +14,7 @@ export const Classes = classes;
 
 export { DateRange, DateRangeBoundary } from "./common/dateUtils";
 export { Months } from "./common/months";
+export { DateFormat, IDateFormatter } from "./dateFormatter";
```

Comment:
~Do we need to export `DateFormat` if we enumerate the union types inline in the props interfaces? Seems like more of a convenience type internally.~

I guess it's used in the `docs-app` package below.